webgl/lessons/ja/webgl-boilerplate.md

```diff
@@ -86,7 +86,7 @@ GPUの特性を活かすため、これらの関数はGPU専用の言語で記
 しかし、ここに挙げた短いコードは、「ほとんど全てのWebGLプログラムで共通」です。
 
 
-私は、シェーダーのコードを&lt;script&gt;タグを使って書く方法を好んで使っています。
+私は、シェーダーのコードを<script>タグを使って書く方法を好んで使っています。
```

Comment:
changes like this shouldn't work (if they are it's a bug?)

markdown allows html so this would need to be  

```
     blablabla`<script>`blablalba
```

not

```
     blablabla<script>blablalba
```